service/history/queueFactoryBase.go

```diff
@@ -45,13 +44,6 @@ import (
 	wcache "go.temporal.io/server/service/history/workflow/cache"
 )
 
-const (
-	QueueFactoryFxGroup = "queueFactory"
```

Comment:
We still need this actually. Can you rebase?

Reply:
👍 Thanks for catching this.